### PR TITLE
Added coverage to the project

### PR DESCRIPTION
**Pull request by rinslow**
it's very low as-of-now, but we will improve our tests for sure :)

Solves https://github.com/Reznic/Inhalator/issues/88

Signed-off-by: rinslow <elkissdan@gmail.com>

### CODE REVIEW

**Comment by rinslow** [comment]
We need to add `coveralls` to the project using @Reznic 's account (he hosts the repo) 
Will be done in a future issue

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,8 @@ language: python
 
 before_install:
   - pip install -r requirements.txt
+  - pip install -r test_requirements.txt
 
 script:
-  - pytest --ignore "tests/unit/drivers/"
-  
\ No newline at end of file
+  - pytest --ignore "tests/unit/drivers/" --cov=.
```
Comment (osherdp) [inline .travis.yml]: add `tox.ini` to the project, and do the actions through it
then the `--cov` and `--ignore` can be applied by default (I think we do it in ipdbugger)

Comment (osherdp) [inline .travis.yml]: delete the ""

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,4 @@ __pycache__/*
 *.log*
 *.service
 config.json
+.coverage
```
Comment (osherdp) [inline .gitignore]: also `htmlcov`, no?

Comment (rinslow) [reply .gitignore]: I really hate `htmlcov` it spams the fuck out of your filesystem,  even though it's useful when actually researching coverage

add it or no?

Comment (osherdp) [reply .gitignore]: You can add it and ignore it in git
Also pycharm can ignore it with a simple action
Alternatively, you can put it under .tox

File: .travis.yml (hunk 1 above)
Comment (osherdp) [reply .travis.yml]: And update README after that

**Pull request merged**